src/ProjectList/defaultProjects.ts

```diff
@@ -34,34 +28,44 @@ export const defaultProjects: Project[] = [
       { title: "Staging", target: "https://stagingapi.artsy.net" },
       { title: "Production Sidekiq", target: "https://api.artsy.net/admin/sidekiq" },
       { title: "Staging Sidekiq", target: "https://stagingapi.artsy.net/admin/sidekiq" },
-      { title: "GitHub", target: "https://github.com/artsy/gravity", primaryAction: true },
-      { title: "CircleCI", target: "https://app.circleci.com/pipelines/github/artsy/gravity", secondaryAction: true },
+      { title: "Sentry Production", target: "https://artsynet.sentry.io/issues/?project=157936" },
+      { title: "Sentry Staging", target: "https://artsynet.sentry.io/issues/?project=157935" },
     ],
   },
   {
     name: "Metaphysics",
     links: [
       { title: "Production", target: "https://metaphysics-production.artsy.net" },
       { title: "Staging", target: "https://metaphysics-staging.artsy.net" },
-      { title: "GitHub", target: "https://github.com/artsy/metaphysics", primaryAction: true },
-      {
-        title: "CircleCI",
-        target: "https://app.circleci.com/pipelines/github/artsy/metaphysics",
-        secondaryAction: true,
-      },
     ],
   },
+  { name: "Palette", links: [] },
+  { name: "Palette-Mobile", links: [] },
   {
     name: "Volt",
     links: [
       { title: "Production", target: "https://cms.artsy.net" },
       { title: "Staging", target: "https://cms-staging.artsy.net" },
-      { title: "GitHub", target: "https://github.com/artsy/volt", primaryAction: true },
+      { title: "Sentry Production", target: "https://artsynet.sentry.io/issues/?project=1281370" },
+      { title: "Sentry Staging", target: "https://artsynet.sentry.io/issues/?project=1281353" },
+    ],
+  },
+  { name: "Vortex", links: [] },
+];
+
+const buildProjects = (projects: Project[]): Project[] =>
+  projects.map(({ name, links }) => ({
+    name,
+    links: [
+      { title: "GitHub", target: `https://github.com/artsy/${name.toLowerCase()}`, primaryAction: true },
+      { title: "GitHub Pull Requests", target: `https://github.com/artsy/${name.toLowerCase()}/pulls` },
       {
         title: "CircleCI",
-        target: "https://app.circleci.com/pipelines/github/artsy/volt",
+        target: `https://app.circleci.com/pipelines/github/artsy/${name.toLowerCase()}`,
```

Comment:
GitHub and CircleCI links can be generated from the repo name. What do you think about building them like this instead of writing the links down for every project?

Reply:
Yeah this is a good idea - I noticed this too but just kept going to get something built. What you have here is better!